src/server/migrations/20201205214200_add_group_id_users_fk.js

```diff
@@ -0,0 +1,11 @@
+exports.up = function (knex) {
+  return knex.schema.table('users', function (table) {
+    table.integer('group_id').unsigned().references('id').inTable('groups');
+  });
+};
+
+exports.down = function (knex) {
+  return knex.schema.table('users', function (table) {
+    table.dropForeign('group_id');
+  });
```

Comment:
You also need to `dropColumn` after dropping the foreign key.

Reply:
dropColumn added to down functions